compile.sh

```diff
@@ -1,11 +1,11 @@
 #|| goto :windows_detected
 
 # Do we already have a repo?
-if [[ -d .git && -f easyinstall.sh && -f src/BlockArea.cpp ]]; then # A good enough indicator that we're in the MCServer git repo.
+if [[ -d .git && -f easyinstall.sh && -f MakeLuaAPI.cmd ]]; then # A good enough indicator that we're in the Cuberite git repo.
```

Comment:
`MakeLuaAPI.cmd` has been removed and I have changed the check to `src/BlockArea.cpp`, why are you reverting my change back?